truss/truss_config.py

```diff
@@ -75,6 +75,38 @@ def from_str(acc_spec: Optional[str]):
         return AcceleratorSpec(accelerator=acc, count=count)
 
 
+@dataclass
+class HuggingFaceModel:
+    repo_id: str = ""
+    revision: str = "main"
```

Comment:
I feel a little antsy about this being the default. I think `Optional[str] = None` would be ideal here, but that would make passing in the revision to the cache wamer slightly harder, but it's doable if you just drop the revision if `== None`